Makefile

```diff
@@ -66,20 +67,10 @@ dev: DEBUG = desktop:*
 dev: 
 	$(MAKE) start NODE_ENV=$(NODE_ENV) DEBUG=$(DEBUG)
 
-
-BASE_CONFIG := $(THIS_DIR)/desktop-config/config-base.json
-ENV_CONFIG := $(THIS_DIR)/desktop-config/config-$(CONFIG_ENV).json
-
-.PHONY: desktop$/config.json
-desktop$/config.json:
-ifeq (,$(wildcard $(ENV_CONFIG)))
-	$(warning Config file for environment "$(CONFIG_ENV)" does not exist. Ignoring environment.)
```

Comment:
I've removed this clause, since `desktop-config/config-%.json` is now a dependent target, so `make` will check for us if the file really exists.